### ISSUE TITLE
[Icon] role="presentation" might be invalid

### ISSUE THREAD

**Issue by ljosberinn** [chakra-ui/chakra-ui#1197]
`<Icon />`s automatically receive the `role="presentation"` attribute which is deemed invalid by the [W3C HTML Validator](https://validator.w3.org/nu/):

![image](https://user-images.githubusercontent.com/29307652/87295309-487ecf80-c505-11ea-834d-1df6bf21e85f.png)

Was the intention to hide the SVG from screenreaders? Because css-tricks mentions adding `role="presentation"` [to elements _within_ an SVG](https://css-tricks.com/accessible-svgs/#conclusion) to hide those in particular.

**Expected Behavior** 
Valid HTML, or rather, no validation error.

**Comment by with-heart** [comment]
Here's the w3c usage spec on `presentation`: https://www.w3.org/TR/using-aria/#ariapresentation

**Comment by segunadebayo** [comment]
@ljosberinn, we do this to prevent us from using only icons to describe certain statuses or labels.

An icon itself doesn't have any semantic meaning.  It's wrapper (button or span) gives it semantic meaning by either using aria-label or providing a visible label.

I think it's valid but we can revisit this later. Thanks for bringing it up.
